.travis.yml

```diff
@@ -17,7 +17,7 @@ install:
 
 script:
   - mkdir -p build/logs
-  - php vendor/bin/phpunit --coverage-clover=build/logs/clover.xml tests
+  - php vendor/bin/phpunit --coverage-clover=build/logs/clover.xml -c tests/phpunit.xml.dist
```

Comment:
I usually `cd` to directory with `phpunit.xml.dst`, but you can keep this implementation because it shows different approach for solving that task, which cool for educational purposes for people, that look on this `.travis.yml` file later.